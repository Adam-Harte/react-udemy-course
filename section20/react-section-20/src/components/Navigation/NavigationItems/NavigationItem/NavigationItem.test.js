import React from 'react';
import renderer from 'react-test-renderer';

import { navigationItem } from './NavigationItem';

describe('<NavigationItem />', () => {
   it('should render an li element and NavLink component to match the snapshot', () => {
      const tree = renderer.create(<navigationItem link="/test" >Test</navigationItem>).toJSON();
      expect(tree).toMatchSnapshot();
   });
});