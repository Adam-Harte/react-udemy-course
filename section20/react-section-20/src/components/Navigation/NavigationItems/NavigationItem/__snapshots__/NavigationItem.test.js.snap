// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationItem /> should render an li element and NavLink component to match the snapshot 1`] = `
<navigationItem
  link="/test"
>
  Test
</navigationItem>
`;
